clients/src/main/java/org/apache/kafka/clients/admin/ListConsumerGroupOffsetsOptions.java

```diff
@@ -44,10 +45,21 @@ public ListConsumerGroupOffsetsOptions topicPartitions(List<TopicPartition> topi
         return this;
     }
 
+    /**
+     * Sets an optional requireStable flag.
+     */
+    public void requireStable(final boolean requireStable) {
+        this.requireStable = requireStable;
+    }
+
     /**
      * Returns a list of topic partitions to add as part of the result.
      */
     public List<TopicPartition> topicPartitions() {
         return topicPartitions;
     }
+
+    public boolean shouldRequireStable() {
```

Comment:
nit: `requireStable`?

Reply:
I intentionally made it so to differentiate with the setter. I saw that in some options we use exactly the same function name, but in some other options we do similar things like here. I'm personally fine either way, will include it in the KIP proposal and let everyone to debate on it.

Reply:
Didn’t we settle on using requireStable in the KIP?